services/portal/studyViewer/studyViewerTasks.js

```diff
@@ -0,0 +1,75 @@
+const studyViewerProps =  require('./studyViewerProps.js');
+const users = require("../../../utils/user");
+const stringify = require('json-stringify-safe');
+
+const I = actor();
+let req_id;
+
+module.exports = {
+
+    goToStudyViewerPage() {
+        I.amOnPage(studyViewerProps.path); ///study-viewer/clinical_trials
+        I.waitForVisible(studyViewerProps.studyViewerDivClass, 5); //.study-viewer
+    },
+
+    goToStudyPage() {
+        I.amOnPage(studyViewerProps.datasetPath);
+        I.waitForVisible(studyViewerProps.datasetDivClass, 5);
```

Comment:
I suggest adding the following line:
```
I.saveScreenshot(`navigating_to_studyViewer_page.png`);
```
To help you troubleshoot the css locator failure.